modules/custom/d_commerce/config/optional/block.block.user_account_menu.yml

```diff
@@ -0,0 +1,24 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - system.menu.account
+  module:
+    - system
+  theme:
+    - droopler_theme
```

Comment:
Czy to zadziała na innych themach?

Reply:
hmm, ja o to już pytałem, te konfigi powinny być wciągniętę w hook_install - tylko, że one są w innym PR - tym drugim :)

Reply:
aaa cofam, tam są inne moduły jednak..

Reply:
Skończę service robić i wtedy go tutaj dodam i zadziała ;)